test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/Utilities/TaskExtensions.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNetCore.Server.Kestrel.FunctionalTests
+{
+    public static class TaskExtensions
```

Comment:
We already have this in https://github.com/aspnet/Testing/blob/dev/src/Microsoft.AspNetCore.Testing/TaskExtensions.cs